src/main/java/io/debezium/connector/vitess/connection/KeyMetaData.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.vitess.connection;
+
+/**
+ * If a column is part of the primary key (including multi-column primary key),
+ * or a column is a single-column unique key
+ */
+public enum KeyMetaData {
+    // The column is part of the primary key (including multi-column primary key)
```

Comment:
Small nit: Could you use the JavaDoc style:

```
/**
  * The column is part of the primary key (including multi-column primary key)
  */
```

That way, the comment will be properly rendered e.g. when hovering over the enum member name in an IDE.

Reply:
Thanks. I didn't know about that. Fixed